app/controllers/users_controller.rb

```diff
@@ -995,18 +995,21 @@ def destroy
     render json: success_json
   end
 
-  def ignore
+  def track
     raise Discourse::NotFound unless SiteSetting.ignore_user_enabled
-    guardian.ensure_can_ignore_user!(params[:ignored_user_id])
 
-    IgnoredUser.find_or_create_by!(user: current_user, ignored_user_id: params[:ignored_user_id])
-    render json: success_json
-  end
-
-  def unignore
-    raise Discourse::NotFound unless SiteSetting.ignore_user_enabled
+    if params[:state] == "ignore"
```

Comment:
This method definitely needs some tests :)

Reply:
Sure 👍 It is in Draft :) Needed to get some opinions first.